kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -68,24 +69,45 @@
         password: '',
         passwordConfirm: '',
         full_name: '',
-        role: 'learner',
+        kind: UserKinds.LEARNER,
```

Comment:
Was curious about this. I'm only using it as a default value, but is this OK? 

I can't use a `computed` value in `data` as far as I can tell, because scoping gets a little funky with `data` [having to be a function](https://vuejs.org/v2/guide/components.html#data-Must-Be-a-Function). From what I can tell, I'd have to create a variable outside of the vue instance that dupes `UserKinds` and then use that.... which seems redundant. 

Speaking from the experience I had with the sorting stuff in `user-page/index`, I can't pass in `this.computed` as the function's default parameter because the closure begins at the function declaration.

Is this OK with you? and if not, any ideas?